config/internal.default.json

```diff
@@ -20,5 +20,6 @@
 	"payment_service": "http://localhost:5000",
 	"internal_service": "http://localhost:3001",
 	"jwt_keys_dir": "jwt",
+	"sign_in_types": ["jwt", "whitelist"],
```

Comment:
i'd feel more comfortable if the default is always `jwt` and you'll need to opt-in for it

Reply:
actually sign_in_type should only be defined for public endpoint, but because the shared model uses it, it has to be defined in both. Mabye I'll pass the global sign_in_type config as a parameter to the model function instead